dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/ListSearchAppsRequest.java

```diff
@@ -26,6 +26,8 @@
  */
 public class ListSearchAppsRequest extends V7<ListSearchApps, ListSearchAppsRequest.Body> {
 
+  private static final int LIMIT = 15;
```

Comment:
Can't remember if we decided changing this, it was updated from 10->15 right?

Reply:
I think initially it was decided 25, but after the decision to not have local filters, I figured I'd drop it down to 15 because when I was testing 25, it seemed too much of a padding and too much apps in one call for nothing.

So yeah, I changed it from 10 -> 15

Reply:
ok, we can leave 15.